src/components/Navigation/Navigation.scss

```diff
@@ -282,14 +285,19 @@ $secondary-item-font-size: rem(15px);
 
     @include breakpoint-after(nav-min-window-corrected()) {
       font-size: $item-font-size-small;
-      line-height: rem(28px);
+      line-height: rem(16px);
```

Comment:
This should probably be unitless `line-height: 1` if the base font size is `16px`